Makefile

```diff
@@ -37,13 +37,21 @@ test-unit:
 # Run the code generation tasks.
 # Example:
 #    make update
-update: install-go-bindata
+update: install-go-bindata install-go-deps
 	hack/update-codegen.sh
 	hack/update-generated-bindata.sh
 
 install-go-bindata:
 	hack/install-go-bindata.sh
 
+install-go-deps:
+	go mod tidy
+	rm -rf ./vendor
+	go mod vendor
+	# go mod does not vendor in scripts so we need to get them manually
```

Comment:
is there any way to whilelist these files in go mod?

Reply:
not that I know of

Reply:
I think if the directory itself was recognized as go module, it would probably respect the files. (some scripts in other modules are also vendored)